examples/expression-oriented-style/expression_oriented_style.bal

```diff
@@ -0,0 +1,31 @@
+import ballerina/io;
+
+// You can define the return value expression using the `=>` notation instead of using
+// curly braces to define the function body block when the function body is just
+// a return statement with an expression.
+function inc1(int x) returns int => x + 1;
+
+// `inc2(int x)` is the same as` inc1(int x)`.
+function inc2(int x) returns int {
+    return x + 1;
+}
+
+var obj = object {
+    private int x = 1;
+    // Same also applies to methods
+    function getX() returns int => self.x;
+};
+
+// Arrow function expression can also be used with query statements
```

Comment:
Is this what we are actually trying to emphasize here?

Reply:
Isn't this for 

> Let expressions (like let clauses in query expressions) allow you to do more with an expression

from the slides?

Reply:
Changed to above.